index.html

```diff
@@ -116,38 +116,49 @@ <h2>A project for first-time contributions</h2>
         <!-- DO NOT modify the TEMPLATE directly, make a copy and paste it below -->
         <!-- make sure there is one line of space above and below your card -->
         <!-- ========== Paste YOUR CARD directly BELOW this line  ========== -->
-         <!-- ________ Puneet's card START ________  -->
+
+
+        
+
+            <!-- ________ JacksonSan009 card START ________  -->
         <div class="card">
-          <p class="name">Puneet Kaur Jattana</p>
+          <p class="name">Jackson Arango</p>
```

Comment:
Hi @JacksonSan009 thanks for the contribution. It seems that while adding your card you have deleted someone else's card. "Puneet Kaur Jattana"'s card has been removed in place of yours. You will need to resubmit the PR, adding your card without removing any of the current cards